go/teams/rotate_hidden_test.go

```diff
@@ -184,3 +187,88 @@ func TestRotateHiddenImplicitAdmin(t *testing.T) {
 	err = team.Rotate(context.TODO(), keybase1.RotationType_HIDDEN)
 	require.NoError(t, err)
 }
+
+// Wait for the BG auditor to finish up, and then we'll make sure that the
+func pollForTrue(t *testing.T, g *libkb.GlobalContext, poller func(i int) bool) {
+	// Hopefully this is enough for slow CI but you never know.
+	wait := 10 * time.Millisecond * libkb.CITimeMultiplier(g)
+	found := false
+	for i := 0; i < 10; i++ {
+		if poller(i) {
+			found = true
+			break
+		}
+		g.Log.Debug("Didn't get an update; waiting %s more", wait)
+		time.Sleep(wait)
+		wait = wait * 2
+	}
+	require.True(t, found, "whether condition was satisfied after polling ended")
+}
+
+func TestHiddenNeedRotate(t *testing.T) {
```

Comment:
worth it to add a  test to make sure it's queued properly in box auditor? can use "RetryNextBoxAudit" to consume one off the queue.

Reply:
So you're saying after the test is over, there will only be my team stuck in the box auditor?

Reply:
Couldn't get this to work!